tests/functional/sponsorship/sponsorship_tests.py

```diff
@@ -58,13 +56,12 @@ def setUpClass(self):
             }
 
     @classmethod
-    def tearDownClass(self):
-        super().tearDownClass()
+    def teardown_class(self):
+        super().teardown_class()
         print(">> removing temporary appconfig")
         os.remove(self.appconfig_loc)
         
-    @tools.nottest
```

Comment:
renamed some functions so that things we actually want collected as tests start in 'test_' and those that we don't want collected as tests do not